include/net/net_context.h

```diff
@@ -106,9 +110,11 @@ typedef void (*net_context_send_cb_t)(struct net_context *context,
  * @typedef net_tcp_accept_cb_t
  * @brief Accept callback
  *
- * @details The accept callback is called after a successful
- * connection is being established or if there was an error
- * while we were waiting for a connection attempt.
+ * @details The accept callback is called after a successful connection is
+ * being established or if there was an error while we were waiting for a
+ * connection attempt. This callback is called by RX thread so its stack and
+ * execution context is used here. Do processing inside this callback as fast
```

Comment:
Keep processing in the callback minimal to reduce the time spent blocked while handling packets.